packages/comms/src/ecl/activity.ts

```diff
@@ -63,6 +63,15 @@ export class Activity extends StateObject<UActivityState, IActivityState> implem
         return this.Running.ActiveWorkunit.filter(awu => clusterName === "" || awu.ClusterName === clusterName).map(awu => Workunit.attach(this.connection.connectionOptions(), awu.Wuid, awu));
     }
 
+    setBanner(request: Partial<WsSMC.Activity>): Promise<Activity> {
```

Comment:
It would be better to specify which items from WsSMC.Activity are needed?

Reply:
I didn't notice that there's also the WsSMC.SetBanner. Or were you thinking a new interface that's more specific than that?